### PR TITLE
Add GitLab-EE (Enterprise Edition) 9.0.0

### PR DESCRIPTION
**Pull request by kwent**
- Synced with GitLab-CE 9.0.0 version 0.1.6
- Tested on my minikube.

### CODE REVIEW

**Comment by kwent** [comment]
Anyone to trigger tests ?

File: stable/gitlab-ee/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# GitLab Community Edition
+
+[GitLab Community Edition](https://about.gitlab.com/) is an application to code, test, and deploy code together. It provides Git repository management with fine grained access controls, code reviews, issue tracking, activity feeds, wikis, and continuous integration. 
+
+## Introduction
+
+This chart stands up a GitLab Community Edition install. This includes:
+
```
Comment (seanknox) [inline stable/gitlab-ee/README.md]: You'll need to replace the Community Edition references with Enterprise Edition.

**Review by seanknox — CHANGES_REQUESTED** [review]

**Comment by kwent** [comment]
@seanknox My bad i thought i replaced everywhere. Fixed 👍

**Comment by seanknox** [comment]
Thanks @kwent! FWIW this PR looks OK to me, but couldn't truly test as I don't have a GitLab EE account.

**Comment by kwent** [comment]
@seanknox Nothing prevents you to install GitLab EE. You don't need a GitLab EE account, you need to put a license otherwise a lot of features are disabled (you can unblock them with a trial license if you really want to https://about.gitlab.com/free-trial/) otherwise GitLabEE without a license is basically GitlabCE.

**Comment by kwent** [comment]
Good to test/merge ? 9.0.1 is already out there...

**Review by seanknox — APPROVED** [review]

**Pull request merged**